src/main/public/locales/de/translations.json

```diff
@@ -5,6 +5,7 @@
   "page_failed_to_fetch": "Die Seite konnte nicht geladen werden",
   "page-editor_no_write_permissions": "Du hast nicht genügend Berechtigungen um diese Seite zu bearbeiten",
```

Comment:
No translation for **page-header_home**?

Reply:
added

Reply:
https://github.com/mbehlendorf/smeagol/blob/52e94945148c5656c94b76834aad3f0f1a45e066/src/main/public/locales/de/translations.json#L7